keyboards/sets3n/zx50/rules.mk

```diff
@@ -0,0 +1,24 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+ SLEEP_LED_ENABLE = no      # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+OLED_DRIVER_ENABLE = yes    # Enables the use of OLED displays
```

Comment:
```suggestion
OLED_ENABLE = yes           # Enables the use of OLED displays
```

Reply:
Hello！I changed this option to `OLED_ENABLE`. The compiled firmware causes the screen to remain unchanged.Do I need to update my compilation environment？

Reply:
Yeah, you may need to update your repo, in that case. 

This covers all of that: https://docs.qmk.fm/#/newbs_git_best_practices